apollo-portal/src/main/resources/static/scripts/controller/UserController.js

```diff
@@ -21,19 +21,86 @@ user_module.controller('UserController',
 function UserController($scope, $window, $translate, toastr, AppUtil, UserService, PermissionService) {
 
     $scope.user = {};
+    $scope.createdUsers = [];
+    $scope.filterUser = [];
+    $scope.status = '1'
+    $scope.showSearchUsernameInput = false
+    $scope.searchKey = ''
+    $scope.changeStatus = changeStatus
+    $scope.toggleUsernameSearchInput = toggleUsernameSearchInput
+    $scope.searchUsers = searchUsers
+    $scope.resetSearchUser = resetSearchUser
 
     initPermission();
 
+    getCreatedUsers();
+
+    toggleUsernameSearchInput();
+
+    searchUsers();
+
+    resetSearchUser();
+
     function initPermission() {
         PermissionService.has_root_permission()
         .then(function (result) {
             $scope.isRootUser = result.hasPermission;
         })
     }
 
+    function getCreatedUsers() {
+        var size = 10;
```

Comment:
I think we can't provide the pagination params as there is no logic to load more users.